package.json

```diff
@@ -16,16 +16,16 @@
     "framer-motion": "^10.16.14",
     "gray-matter": "^4.0.3",
     "highlight.js": "^11.9.0",
-    "next": "^14.0.3",
+    "next": "^14.1.1-canary.66",
```

Comment:
```
   Creating an optimized production build ...
 ⨯ Failed to find font override values for font `Young Serif`
 ```
 
 Fixed in the Next.js `14.1.1-canary.66`
 
 - https://github.com/vercel/next.js/issues/47115